source/extensions/filters/http/alternate_protocols_cache/BUILD

```diff
@@ -0,0 +1,43 @@
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_extension",
+    "envoy_cc_library",
+    "envoy_extension_package",
+)
+
+licenses(["notice"])  # Apache 2
+
+envoy_extension_package()
+
+envoy_cc_library(
+    name = "filter_lib",
+    srcs = ["filter.cc"],
+    hdrs = ["filter.h"],
+    visibility = ["//visibility:public"],
```

Comment:
why public?  by default extensions shouldn't need this

Reply:
Done.